### DISCUSSION TITLE
Sounds for wand connected

### DISCUSSION THREAD

**Discussion by Toy203** [gpstar81/GPStar-proton-pack#330] [Ideas]
Could the pack make some kind of electrical sound once the wand cable is connected? Could be cool to have it.

**Comment by Toy203** [comment]
Addendum: afterlife should sound like a shotgun cocking, FE like a pistol lol

**Comment by gpstar81** [comment]
> Could the pack make some kind of electrical sound once the wand cable is connected? Could be cool to have it.

Good idea.
